packages/main/src/RPA/Browser/Selenium.py

```diff
@@ -54,10 +56,73 @@ def suppress_logging():
         logger.warning, logger.error = logger_warning, logger_error
 
 
+def ensure_scheme(url, default):
+    """Ensures that a URL has a scheme, such as `http` or `https`"""
+    if default is None:
+        return url
+
+    parts = list(urllib.parse.urlsplit(url))
+    if not parts[0]:
+        parts[0] = str(default)
+        url = urllib.parse.urlunsplit(parts)
+
+    return url
+
+
 class BrowserNotFoundError(ValueError):
     """Raised when browser can't be initialized."""
 
 
+class BrowserManagementKeywordsOverride(BrowserManagementKeywords):
+    """Overriden keywords for browser management."""
+
+    def __init__(self, ctx):
+        super().__init__(ctx)
+        self._default_scheme = "https"
+
+    @keyword
+    def set_default_url_scheme(self, scheme: Optional[str]) -> None:
+        """Sets the default `scheme` used for URLs without a defined
+        value, such as `http` or `https`.
+        """
+        self._default_scheme = scheme
+
+    @keyword
+    def go_to(self, url: str) -> None:
+        url = ensure_scheme(url, self._default_scheme)
+        super().go_to(url)
+
+    go_to.__doc__ = BrowserManagementKeywords.go_to.__doc__
```

Comment:
Does its job, but if you feel this is even more elegant, then go for it (not tested):
```python
@keyword
@functools.wraps(BrowserManagementKeywords.go_to)
def go_to(...)
```

Reply:
functools.wraps effectively makes the outer look like the inner, but maybe a bit too much in this inheritance case. it also replaces attributes like `__qualname__` and `__module__`, and I'm not confident that won't have some surprising side-effects.